pkg/egressgateway/policy.go

```diff
@@ -17,11 +17,20 @@ import (
 	k8sLabels "github.com/cilium/cilium/pkg/k8s/slim/k8s/apis/labels"
 	slim_metav1 "github.com/cilium/cilium/pkg/k8s/slim/k8s/apis/meta/v1"
 	"github.com/cilium/cilium/pkg/logging/logfields"
+	"github.com/cilium/cilium/pkg/maps/egressmap"
 	nodeTypes "github.com/cilium/cilium/pkg/node/types"
 	"github.com/cilium/cilium/pkg/policy"
 	"github.com/cilium/cilium/pkg/policy/api"
 )
 
+// PolicyMap is used to communicate EGW policies to the datapath.
+type PolicyMap interface {
```

Comment:
This could live in `egressmap`, but then we hide the concrete implementation of the policy map by default. That gets annoying if some other part of the code base needs another method, which then forces egressgw mocks / fakes to implement that method as well.

Happy to move this around if someone feels strongly about it.

Reply:
Ended up moving this, testing becomes a mess otherwise...